PLAYER/SetPlayerUnderwaterTimeRemaining.md

```diff
@@ -2,15 +2,20 @@
 ns: PLAYER
 aliases: ["0xA0D3E4F7AAFB7E78"]
 ---
-## _SET_PLAYER_UNDERWATER_TIME_REMAINING
```

Comment:
Since this native doesn't have an original Joaat Hash, you will not be able to name the native without prefixing it with `_`.